config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -144,8 +143,6 @@ periodics:
   cluster: k8s-infra-prow-build
   interval: 12h
   decorate: true
-  skip_branches:
-  - release-\d+\.\d+ # per-release settings
```

Comment:
I'm sure this is a valid option

Reply:
it doesnt exist as an option

Reply:
you can check the struct as well as the documentation

Reply:
yeah, because it is a periodic. The branch it runs on is implicit from the extra_ref[*].base_ref

Reply:
`{"component":"checkconfig","file":"k8s.io/test-infra/prow/cmd/checkconfig/main.go:91","func":"main.reportWarning","level":"warning","msg":"error unmarshaling config/jobs/kubernetes/sig-testing/conformance-e2e.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"skip_branches\"","severity":"warning","time":"2022-09-08T17:29:24Z"}
`